.github/workflows/pre-commit-update.yml

```
@@ -0,0 +1,30 @@
+name: Pre-commit auto-update
+on:
+  schedule:
+    # Run on mondays
+    - cron: "0 0 * * 1"
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Please pin all action versions to the full semantic version.
```suggestion
      - uses: actions/checkout@v2.0.0
```